conf/content_host.yaml.template

```diff
@@ -2,51 +2,51 @@ content_host:
   default_rhel_version: 7
   rhel6:
     vm:
-      workflow: deploy-base-rhel
+      workflow: deploy-rhel
       deploy_rhel_version: '6'
       target_memory: 1GiB
```

Comment:
Please note, at this time (may be changing soon), the values for `target_memory` and `target_cores` do nothing

Reply:
Yeah its a old config template and can be updated. BTW why are we going away from memory and cores conf? cause we Do have profiles to choose ?

Reply:
You currently cannot select target_*. You choose a flavor only.

Reply:
BTW Not a blocker for this MR, so once `may be changing soon` is confirmed, we will change this in the template.